packages/unmock-node/src/loaders/fs-service-def-loader.ts

```diff
@@ -0,0 +1,89 @@
+import debug from "debug";
+import fs from "fs";
+import path from "path";
+import { IServiceDef, IServiceDefLoader } from "unmock-core";
+
+const DEFAULT_SERVICE_SUBDIRECTORY = "__unmock__";
+
+export interface IFsServiceDefLoaderOptions {
+  servicesDir?: string;
+}
+
+const debugLog = debug("unmock:fs-service-def-loader");
```

Comment:
This feels a bit weird as we haven't used `debug` at all so far in the project.
If we decide to use it, we should probably use it everywhere?

Or is this a leftover?

Reply:
I think we definitely need some logging and `debug` is the standard for that, I think?

Reply:
I don't doubt that - we just haven't used `debug` elsewhere yet.